packages/chaire-lib-backend/src/services/auth/userEmailNotifications.ts

```diff
@@ -95,6 +103,16 @@ export const sendConfirmationEmail = async (
     console.log('Email sent for account confirmation');
 };
 
+export const sendConfirmedByAdminEmail = async (user: UserModel): Promise<void> => {
+    const email = {
+        mailText: 'server:confirmedByAdminEmailText',
+        mailSubject: 'server:confirmedByAdminEmailSubject',
+        toUser: user
+    };
+    await sendEmail(email, {});
+    console.log('Email sent confirmation by admin');
```

Comment:
Email sent for* confirmation. Otherwise it's the email who sent a confirmation by admin, doesn't make sense :p